src/_P036_FrameOLED.ino

```diff
@@ -298,6 +397,10 @@ boolean Plugin_036(byte function, struct EventStruct *event, String& string)
         }
         if (UserVar[event->BaseVarIndex] == 1) {
           // Display is on.
+          OLEDIndex = PCONFIG(8);
+          HeaderContent = eHeaderContent(PCONFIG(9));
```

Comment:
There are only 8 values for PCONFIG.
So you're altering settings of the "next" task.